app/send-team-notification.js

```diff
@@ -35,116 +37,93 @@ exports.sendTeamNotification = async function sendNotification(teamConfig, repor
+            for (let index = 0; index < webhookURL.records.length; index++) {
+                message.push(webhookURL.records[index].config.webhookName)
+                await axios.post(webhookURL.records[index].config.webhookURL, config)
+                    .then(async (res) => {
+                        try {
+                            if (res.data == "1") {
+                                notificationSent = true;
+                            }
+                            else {
+                                errorMsg = res.data;
+                                notificationSent = false;
+                            }
+
+                        } catch (error) {
+                            errorMsg = error;
+                            logger.log({
+                                level: 'error',
+                                message: 'error occurred while sending team' + reportData.report_obj.thresholdAlert ? ' for threshold alert' : '',
+                                errMsg: error,
+                            });
+
+                            reject({
+                                success: 0,
+                                message: error
+                            });
                         }
-
-
-                    } catch (error) {
-                        transaction.rollback();
+                    })
+                    .catch((error) => {
                         logger.log({
                             level: 'error',
-                            message: 'error occurred while sending team' + reportData.report_obj.thresholdAlert ? ' for threshold alert' : '',
+                            message: 'error occurred while sending team message ' + reportData.report_obj.thresholdAlert ? ' for threshold alert' : '',
                             errMsg: error,
                         });
-                    }
-                })
-                .catch((error) => {
-                    logger.log({
-                        level: 'error',
-                        message: 'error occurred while sending team message ' + reportData.report_obj.thresholdAlert ? ' for threshold alert' : '',
-                        errMsg: error,
-                    });
-                    let shedularlog = models.SchedulerTaskLog.create({
-                        SchedulerJobId: reportData['report_shedular_obj']['id'],
-                        task_executed: new Date(Date.now()).toISOString(),
-                        task_status: "team " + error,
-                        thresholdMet: reportData.report_obj.thresholdAlert,
-                        notificationSent: false,
-                        channel: 'Teams'
-                    });
-                })
-        }
 
-        logger.log({
-            level: 'info',
-            message: 'team message send ' + reportData.report_obj.thresholdAlert ? ' for threshold alert' : ''
-        });
+                        reject({
+                            success: 0,
```

Comment:
place 0,1 status code in separate js file, rather use the standard codes 

https://restfulapi.net/http-status-codes/